pkg/cmd/roachprod/main.go

```diff
@@ -668,6 +668,36 @@ non-terminating signal (e.g. SIGHUP), unless you also configure --max-wait.
 	}),
 }
 
+var deployCmd = &cobra.Command{
+	Use:   "deploy <cluster> <application> <version>",
+	Short: "deploy a new version of cockroach",
+	Long: `Performs a rolling update, to avoid down time, of cockroach.
```

Comment:
Nit: `upgrade`?

What's the "to avoid downtime" part about?

Reply:
Will change to upgrade. The to avoid down time refers to the "rolling" part, as in to: down and upgrade binary, and restart nodes one at a time. It's probably obvious and superfluous, I'll remove the statement. All of this is implied by rolling, which is enough to tell us this doesn't stop the whole cluster, upgrades the binary and then restarts the whole cluster all at once.